### ISSUE TITLE
Can't see custom emotion correctly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#322]
```
What steps will reproduce the problem?
1. That log-in MSN by this plug-in can't see custom emotion correctly.

What is the expected output? What do you see instead?
Original Pidgin can receive MSN custom emotion correctly. I wanna the pecan 
also fix the problem.

Please provide any additional information below.
0.1.3
```

Original issue reported on code.google.com by `s901510...@gmail.com` on 6 Nov 2011 at 10:10
- Blocked on: #320
- Merged into: #311

**Comment by GoogleCodeExporter** [comment]
Original comment by `devid...@gmail.com` on 14 Nov 2011 at 4:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `devid...@gmail.com` on 14 Nov 2011 at 4:59
- Changed state: **Duplicate**
